core/tx_pool.go

```diff
@@ -653,7 +653,7 @@ func (pool *TxPool) add(tx *types.Transaction, local bool) (bool, error) {
 		log.Trace("Pooled new executable transaction", "hash", hash, "from", from, "to", tx.To())
 
 		// We've directly injected a replacement transaction, notify subsystems
-		go pool.txFeed.Send(TxsPreEvent{types.Transactions{tx}})
+		go pool.txFeed.Send(NewTxsEvent{types.Transactions{tx}})
```

Comment:
Will it be better if we post the `NewTxsEvent` synchronously here?

Reply:
Felix started looking into the synchronous postings, so I'd rather keep that can of worms in a separate pr.